book/src/connecting/connecting.md

```diff
@@ -30,4 +30,15 @@ async fn main() -> Result<(), Box<dyn Error>> {
 ```
 
 After succesfully connecting to some specified node the driver will fetch topology information about
-other nodes in this cluster and connect to them as well.
\ No newline at end of file
+other nodes in this cluster and connect to them as well.
+
+```eval_rst
```

Comment:
@cvybhu could you review and verify if these additions interact with book generation? Will mdbook be able to process them without issues?